### PR TITLE
Updates build.gradlde 

### LINKED ISSUE #206: Does this Plugin support Android X now?
As the title said ,i don't know if this plugin support android X ,someone could help me?

### PR DESCRIPTION
**Pull request by markflarup**
to include mavenCentral() which makes me able to include this package in Android projects. Furthermore, some minor updates has been updated to the latest version. This fixes #206.

### CODE REVIEW

**Comment by mikes222** [comment]
After 3 hours of hard work the app is starting finally. Please change the following files in addition to this pull-request:

`example/android/build.gradle`:
        `classpath 'com.android.tools.build:gradle:3.4.0'`

`example/android/gradle/wrapper/gradle-wrapper.properties`:
    `distributionUrl=https\://services.gradle.org/distributions/gradle-5.1.1-all.zip`

I see now that the 2 files should only affect the example app but it should be done anyway.

**Comment by pauldemarco** [comment]
Hi @markflarup and @mikes222,

I typically wait for the official flutter plugins repo to upgrade to newer versions of gradle before making the move with this plugin. I'd be happy to merge this PR though when the time comes.

Thanks.

**Comment by pauldemarco** [comment]
@markflarup @mikes222 I folding in some changes that I overlooked from the official plugin repos.
Let me know if they work for you.

Thanks!

File: example/android/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.1.1-all.zip
```
Comment (andretietz) [inline example/android/gradle/wrapper/gradle-wrapper.properties]:
```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
```

File: example/android/build.gradle [hunk 2]
```diff
@@ -5,7 +5,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.3.0'
+        classpath 'com.android.tools.build:gradle:3.4.0'
```
Comment (andretietz) [inline example/android/build.gradle]:
```suggestion
        classpath 'com.android.tools.build:gradle:3.5.0'
```

File: android/build.gradle [hunk 3]
```diff
@@ -5,11 +5,12 @@ buildscript {
     repositories {
         google()
         jcenter()
+        mavenCentral()
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.3.0'
-        classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.7'
+        classpath 'com.android.tools.build:gradle:3.4.0'
```
Comment (andretietz) [inline android/build.gradle]:
```suggestion
        classpath 'com.android.tools.build:gradle:3.5.0'
```

File: android/build.gradle [hunk 4]
```diff
@@ -5,11 +5,12 @@ buildscript {
     repositories {
         google()
         jcenter()
+        mavenCentral()
```
Comment (andretietz) [inline android/build.gradle]:
```suggestion
        jcenter()
```

File: android/build.gradle [hunk 5]
```diff
@@ -5,11 +5,12 @@ buildscript {
     repositories {
         google()
         jcenter()
```
Comment (andretietz) [inline android/build.gradle]:
```suggestion
        mavenCentral()
```

Comment (andretietz) [reply android/build.gradle]: Always mavenCentral before jcenter, due to fraud cases.

Nice article
https://blog.autsoft.hu/a-confusing-dependency/

Comment (markflarup) [reply android/build.gradle]: I didn't know that! Thanks for enlightening me 👍

**Comment by pauldemarco** [comment]
Recently updated Android and iOS project templates, putting this way out of date.

**Pull request closed without merging**